pkg/interact/telegram.go

```diff
@@ -158,7 +171,24 @@ func (tm *Telegram) Start(context.Context) {
 		if reply.set {
 			reply.build()
 			if len(reply.message) > 0 || reply.menu != nil {
-				checkSendErr(tm.Bot.Send(m.Chat, reply.message, reply.menu))
+				var left, right int
+				for left, right = 0, maxMessageSize; right < len(reply.message); left, right = right, right+maxMessageSize {
```

Comment:
for the split thing, can you pull out it to a function?

Reply:
done